src/main/scala/IsRetweetClassifier.scala

```diff
@@ -0,0 +1,84 @@
+import org.apache.spark.ml.Pipeline
+import org.apache.spark.ml.classification.LogisticRegression
+import org.apache.spark.sql.types._
+import org.apache.spark.sql.{Row, SparkSession}
+import org.apache.spark.ml.feature.{HashingTF, Tokenizer}
+import org.apache.spark.sql.functions._
+
+object IsRetweetClassifier {
+  def main(args: Array[String]) {
+    val spark = SparkSession
+      .builder()
+      .appName("Pipeline")
+      .master("yarn")
+      .getOrCreate()
+
+    import spark.sqlContext.implicits._
+
+    val text = spark
+      .sparkContext.wholeTextFiles("hdfs:///user/ilos/tweets/tweets.txt")
+      .map { case (filename, content) => content}
+
+    val tweets = text
+      .flatMap(line => line.split("List\\("))
```

Comment:
I use saveAsTextFiles method of my DStream object for saving tweets to hdfs. And my file has format List(text of tweet, language of tweet, isRetweet flag). And because text of tweet may content any symbols I can't use simple parsing methods in this case. 
It's the best way that I could find.